types/webxr/index.d.ts

```diff
@@ -663,6 +663,19 @@ interface XRPlane {
 
 declare abstract class XRPlane implements XRPlane {}
 
+// Mesh detection
+type XRMeshSet = Set<XRMesh>;
+
+interface XRMesh {
+	meshSpace: XRSpace;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/webxr/index.d.ts;
  Line: 670;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.